src/screens/Customers/EditCustomerMeter.tsx

```diff
@@ -46,53 +53,54 @@ function EditCustomerMeter() {
 
     const handleSubmit = (values: any) => {
         const { meterNumber, meterType } = values;
-        history.push('tariff-plans', { meterNumber, meterType, goBack: -2 })
+        const meterTypeKey: string = revMeterTypeMap.get(meterType) as string;
+        history.push('tariff-plans', { meterNumber, meterTypeKey, goBack: -2 })
     }
 
     return (
-        <BaseScreen title="Edit Customer" leftIcon="backNav">
+        <BaseScreen title={intl(words.edit_customer)} leftIcon="backNav">
             <form noValidate onSubmit={formik.handleSubmit}>
 
                 <FormControl
                     required
                     variant="outlined"
                     style={{ width: '100%' }}
```

Comment:
Not technically in scope for this PR, but the `Select` box is cut off and a `marginTop` needs to be added to fix it. Can you change the inline style of `FormControl` to `style={{ width: '100%', marginTop: 8, marginBottom: 8 }}`.

Here's a before & after of what it should look like:

### Before:
![image](https://user-images.githubusercontent.com/35501399/116831795-5837d500-ab66-11eb-88d2-52623f0e8f89.png)


### After:
![image](https://user-images.githubusercontent.com/35501399/116831792-4c4c1300-ab66-11eb-9ea0-98d3db2e8b99.png)